indigo_app/templates/indigo_api/task_detail.html

```diff
@@ -141,18 +141,32 @@ <h6>Labels</h6>
         {% endwith %}
       </ul>
 
-      <h6>Workflows</h6>
-      <ul class="list-unstyled">
+
+      <form action="{% url 'task_workflows' place=place.place_code pk=task.pk %}" method="POST" data-submit="ajax">
+        {% csrf_token %}
+
+        <select class="selectpicker" multiple name="workflows" onchange="$(this.form).trigger('submit')"
+            data-dropdown-align-right="true" data-style="btn-link pl-0 text-dark"
+            {% if not perms.indigo_api.change_task %}disabled{% endif %}
+            data-none-selected-text="Workflows" data-selected-text-format="static">
+
+          {% for workflow in possible_workflows|dictsort:'title' %}
+            <option value="{{ workflow.id }}" {% if workflow in task.workflows.all %}selected{% endif %}>{{ workflow.title }}</option>
+          {% endfor %}
+        </select>
+      </form>
+      <ul class="list-unstyled mb-2">
         {% with task.workflows.all as workflows %}
         {% if workflows %}
-          {% for workflow in workflows %}
-            <li><a href="{% url 'workflow_detail' place=place.place_code pk=workflow.pk %}">{{ workflow.title }}</a></li>
+          {% for workflow in workflows|dictsort:'title' %}
+            <li><a href="{% url 'workflow_detail' place=place.place_code pk=workflow.pk %}">#{{ workflow.pk }} - {{ workflow.title }}</a></li>
```

Comment:
```suggestion
            <li><a href="{% url 'workflow_detail' place=place.place_code pk=workflow.pk %}">#{{ workflow.pk }} – {{ workflow.title }}</a></li>
```
Also, why include workflow ids here (and not in dropdown)? (Not sure they need to be included at all?)

Reply:
Good point, I think we should have them everywhere. It makes it much easier to refer to them unambiguously, since it looks like we're going to have many open for a place at once. Same reason we expose the task id.